standard_plus_issue_studio/tests/test_standard_plus_issue.py

```diff
@@ -33,11 +33,10 @@ def setUp(self):
     def test_action_set_submitted(self):
         attachment_count = self.AttachmentObj.search_count([])
         self.assertEqual(self.issue.state, 'draft')
+        self.assertFalse(self.issue.studio_retrieval_date)
         self.issue.action_set_submitted()
         self.assertTrue(self.issue.studio_module)
-        self.assertEqual(
-            self.issue.studio_retrieval_date[:-2], fields.Datetime.now()[:-2]
-        )
+        assert self.issue.studio_retrieval_date
```

Comment:
@rimbendhaou Why not `self.assertTrue(self.issue.studio_retrieval_date)` ?

Reply:
@joaoalf It is the same thing i think?